test/features/caseView.feature

```diff
@@ -27,5 +25,14 @@ Feature: Set of scenarios testing case view functionality
       | Text Field 3        |
       | Text Field 4        |
 
+  @functional
   Scenario: print button visible on case view
```

Comment:
`print button visible on case view when configured` would be more descriptive title please

Reply:
Done.